[ADD] hr_employee_substitute.rst (#23)
focusate/documentation-user: hr/customizations/hr_employee_substitute.rst

===================
Employee Substitute
===================

Allows to specify substitute employees for other employee and select specific
date range, when substitution will be valid.

Configuration
=============

To start using this new functionality, module :code:`hr_employee_substitute`
must be installed.

By default datetime type is used for start/end of substitute, but it is
possible to simplify it and use date type, by checking Date Type for
Substitutes. This option can be enabled in Employees Settings
(:menuselection:`Employees --> Configuration --> Settings`), Under Work

> Organization section by changing value of parameter 'Date Type for Substitutes':

Reviewer: ```suggestion
Organization section by checking 'Date Type for Substitutes' option:
```

.. image:: media/hr_employee_substitute_settings.png
    :align: center

Functionality
=============

This feature allows to specify substitute employees for other employee and
select specific date range, when substitution will be valid.

> In order to view and create employee substitutes navigate to

Reviewer: ```suggestion
In order to view and create employee substitutes, navigate to
```

(:menuselection:`Employees --> Substitutes`)

Here you can see all substitutes in a tree view:

.. image:: media/hr_employee_substitutes_tree_view.png
    :align: center

> In order to create substitutes choose employee, substitute and start/end date

Reviewer: ```suggestion
In order to create substitutes, choose employee, substitute and start/end date
```
